Automattic/jetpack
# Dashboard: hide the Scan card on Atomic sites.

#### Changes proposed in this Pull Request:
﻿
Security scanning is handled by the Atomic platform, so site owners do not need to see the card on their sites on WoA.

#### Jetpack product discussion

Related: p9F6qB-8n9-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Switch to that branch on a WoA site.
* Go to Jetpack > Dashboard.
* You should not see a Scan card.
